### PR TITLE
Bug 1213954 - Add accessibility input panel with touch & hold delay o…

### PR DESCRIPTION
**Pull request by eeejay**
…ption.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dfd282aa0cf4fb8f34da661c98c3f6501d60b2d9

File: apps/settings/locales/settings.en-US.properties [hunk 1]
```diff
@@ -1389,6 +1389,13 @@ screenReader-confirm-button-stop = Stop
 colorFilter-invert=Invert
 colorFilter-grayscale=Grayscale
 colorFilter-contrast=Contrast
+# Device :: Accessibility :: Input
```
Comment (yzen) [inline apps/settings/locales/settings.en-US.properties]: Nit: no spaces around =

File: apps/settings/locales/settings.en-US.properties [hunk 2]
```diff
@@ -1389,6 +1389,13 @@ screenReader-confirm-button-stop = Stop
...
+accessibility-input = Input
+accessibility-input-desc = Touch and gesture accomodations
```
Comment (yzen) [inline apps/settings/locales/settings.en-US.properties]: nit: I would replace 'accomodations' with something shorter and simpler?

Comment (eeejay) [inline apps/settings/locales/settings.en-US.properties]: adjustments?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e7da68e9c0c1c605aac9106634c82d66830e93f

File: apps/settings/locales/settings.en-US.properties (hunk 1 above)
Comment (eeejay) [inline apps/settings/locales/settings.en-US.properties]: Done.

**Pull request merged**